libs/common-services/src/main/java/uk/gov/di/ipv/core/library/domain/UserIdentity.java

```diff
@@ -28,6 +29,7 @@
 @Setter
 @ExcludeFromGeneratedCoverageReport
 @Builder
+@JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
This is a change to the output, but I'm hoping it won't affect any consumers and should help streamline our responses.

(If we don't do this, we have to allow `null` as a valid value for the property in the schemas, which is a bit of a pain!)

cc @CarlyG55